HTML-TAGS/all_tags.html

```diff
@@ -0,0 +1,73 @@
+<! -- In this i have performed all the tags which i have Learned -->
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Brown Bears</title>
+    </head>
+    <body>
+        <div>
+            <table>
+                <tr>
+                    <th>first_name</th>
+                    <th>last_name</th>
+                    <th>Marks</th>
+                </tr>
+                <tr>
+                    <td>Ashutosh</td>
+                    <td>Yadav</td>
+                    <td>100</td>
+                </tr>
+                <tr>
+                    <td>Ashutosh</td>
+                    <td>Yadav</td>
+                    <td>100</td>
+                </tr>
+                <tr>
+                    <td>Ashutosh</td>
+                    <td>Yadav</td>
+                    <td>100</td>
+                </tr>
+            </table>
+        </div>
+        <h1>In this I,Ashutosh Yadav have performed all the tags which i have Learnt</h1>
+        <nav>
+            <ul>
+                <li><a href="#introduction">Introduction</a></li>
+                <li><a href="#habitat">Habitat</a></li>
+                <li><a href="#media">Media</a></li>
+            </ul>
+        </nav>
+        <div id="introduction">
+            <h2>About Brown Bears</h2>
+            <p>
+                The brown bear
+            </p>
+            <a href="https://en.wikipedia.org/wiki/Brown_bear" target="_blank">Learn More</a>
+            <h3>Species</h3>
+            <ul>
+                <li>Arctos</li>
+                <li>Collarus</li>
+                <li>Horribilis</li>
+                <li>Nelsoni (extinct)</li>
+            </ul>
+            <h3>Features</h3>
+            <p>
+                Brown bears are not always completely brown.
+            </p>
+        </div>
+        <div id="habitat">
+            <h2>Habitat</h2>
+            <h3>Countries with Large Brown Bear Populations</h3>
+            <ol>
+                <li>Russia</li>
+                <li>United States</li>
+                <li>Canada</li>
+            </ol>
+        </div>
+        <div id="media">
+            <h2>Media</h2>
+            <img src="https://content.codecademy.com/courses/web-101/web101-image_brownbear.jpg" />
+            <video src="https://content.codecademy.com/courses/freelance-1/unit-1/lesson-2/htmlcss1-vid_brown-bear.mp4" height="240" width="320" controls>Video not supported</video>
+        </div>
+    </body>
+</html>
```

Comment:
Add the doc file in the PR description
Add a new line at the end of file
Delete unnecessary hello_world.txt